Makefile

```diff
@@ -1,27 +1,68 @@
 
+ifndef VERBOSE
+.SILENT:
+endif
+
 .PHONY: all
 all:
 	@echo "No default action."
 
-# docker:
+.PHONY: check
+check:
+	npm run lint
+
+.PHONY: clean
+clean: script-clean-dirs
+
+.PHONY: build
+build: script-build-dirs
 
-docker_subtag := dev-$(shell git describe --tags --dirty)
-docker_tag := hathor-wallet-service:$(docker_subtag)
-docker_build_arg :=
-docker_build_flags :=
-ifneq ($(docker_build_arg),)
-	docker_build_flags +=  --build-arg $(docker_build_arg)
+.PHONY: script-build-dirs
+script-build-dirs:
+ifneq ("$(HEADLESS_SCRIPTS_SKIP_BUILD)", "1")
+	# Building scripts...
+	if [ -d "./dist" ]; then rm -rf ./dist; fi
+	if [ -d "./dist-scripts" ]; then rm -rf ./dist-scripts; fi
+	npm run --silent build > '/dev/null' 2>&1
+	npm run --silent build-scripts > '/dev/null' 2>&1
 endif
```

Comment:
This command removes the current build and builds the source code, but only if `HEADLESS_SCRIPTS_SKIP_BUILD` is **not** `"1"`, since this is defined on the Dockerfile the container will not run this.